tests/responses/all_customers.json

```diff
@@ -0,0 +1,92 @@
+                "mode": "test",
+                "name": "Customer A",
+                "email": "customer@example.org",
+                "locale": "nl_NL",
+                "metadata": null,
+                "recentlyUsedMethods": [
+                    "creditcard",
+                    "ideal"
+                ],
+                "createdAt": "2018-04-06T13:23:21.0Z",
+                "_links": {
+                    "self": {
+                        "href": "https://api.mollie.com/v2/customers/cst_kEn1PlbGa",
+                        "type": "application/hal+json"
+                    },
+                    "documentation": {
+                        "href": "https://docs.mollie.com/reference/v2/customers-api/get-customer",
+                        "type": "text/html"
+                    }
+                }
+            },
+            {
+                "resource": "customer",
+                "id": "cst_kEn1PlbGb",
+                "mode": "test",
+                "name": "Customer B",
+                "email": "customerb@example.org",
+                "locale": "nl_NL",
+                "metadata": null,
+                "recentlyUsedMethods": [
+                    "creditcard",
+                    "ideal"
+                ],
+                "createdAt": "2018-04-06T13:23:21.0Z",
+                "_links": {
+                    "self": {
+                        "href": "https://api.mollie.com/v2/customers/cst_kEn1PlbGa",
+                        "type": "application/hal+json"
+                    },
+                    "documentation": {
+                        "href": "https://docs.mollie.com/reference/v2/customers-api/get-customer",
+                        "type": "text/html"
+                    }
+                } },
+            {
+                "resource": "customer",
+                "id": "cst_kEn1PlbGc",
+                "mode": "test",
+                "name": "Customer C",
+                "email": "customerc@example.org",
+                "locale": "nl_NL",
+                "metadata": null,
+                "recentlyUsedMethods": [
+                    "creditcard",
+                    "ideal"
+                ],
+                "createdAt": "2018-04-06T13:23:21.0Z",
+                "_links": {
+                    "self": {
+                        "href": "https://api.mollie.com/v2/customers/cst_kEn1PlbGa",
+                        "type": "application/hal+json"
+                    },
+                    "documentation": {
+                        "href": "https://docs.mollie.com/reference/v2/customers-api/get-customer",
+                        "type": "text/html"
+                    }
+                }}
+        ]
+    },
+    "_links": {
+        "self": {
+            "href": "https://api.mollie.com/v2/customers",
+            "type": "application/hal+json"
+        },
+        "previous": null,
+        "next": {
+            "href": "https://api.mollie.com/v2/customers?from=cst_stTC2WHAuS",
```

Comment:
does this match the behaviour of `count`? does `count` reflect all results, or only the ones in this paginated result set?

Reply:
Count reflects the amount of results requested